Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1714,12 +1714,30 @@ void AXObjectCache::onPageActivityStateChange(OptionSet<ActivityState> newState)
 #endif
 }
 
-void AXObjectCache::onFocusChange(Node* oldNode, Node* newNode)
+static bool shouldDeferFocusNodeChange(Element* element)
```

Comment:
Sorry, I should've noticed this earlier, but IMO this function name could be a bit more concise, e.g. `shouldDeferFocusChange`.